files/zh-cn/web/api/webgl_api/matrix_math_for_the_web/index.md

```diff
@@ -14,25 +14,15 @@ slug: Web/API/WebGL_API/Matrix_math_for_the_web
 虽然存在许多类型的矩阵，但我们感兴趣的是三维变换矩阵。这种矩阵由一个 4x4 方阵，共 16 个值组成。在 JavaScript 中，可以很方便的用数组表示矩阵。比如典型的单位矩阵。单位阵乘上一个点或者矩阵，其结果保持不变。
 
 ```js
-var identityMatrix = [
-  1, 0, 0, 0,
-  0, 1, 0, 0,
-  0, 0, 1, 0,
-  0, 0, 0, 1
-];
+var identityMatrix = [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1];
 ```
 
 说到乘法，这种运算用于矩阵是什么样的呢？最简单的例子是矩阵乘一个点。你可能注意到，三维空间中的点和一个 4x4 矩阵并不匹配，为此我们加上了额外的第四维 W。一般来说，把 W 设为 1 就可以了。W 维度还有一些额外的用途超出本文的讨论范围。查看[WebGL model view projection](/zh-CN/docs/Web/API/WebGL_API/WebGL_model_view_projection)看它有哪些用途。
 
 注意矩阵和点的对齐方式：
 
 ```js
-[1, 0, 0, 0,
- 0, 1, 0, 0,
- 0, 0, 1, 0,
- 0, 0, 0, 1]
-
-[4, 3, 2, 1]
+[1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1][(4, 3, 2, 1)];
```

Comment:
Maybe we should revert this change. Also in [mdn/content](https://github.com/mdn/content/blob/93ee96f21e4709872b2f3ae9ddb185b350b5e769/files/en-us/web/api/webgl_api/matrix_math_for_the_web/index.md?plain=1#L25-L33)

Reply:
filled: mdn/content#28266